### ISSUE TITLE
Errors at --as step

### ISSUE THREAD

**Issue by pieterlukasse** [opentargets/platform#482]
Spotted this error in the --as step. It states "invalid version format". Is this referring to the .json format? If so, shouldn't the --val step catch this? 

```
2019-03-12 17:29:54,835 mrtarget.common.Redis ERROR - Error processing job b6a82577027147c5bb154a9105f18771: 
Traceback (most recent call last):
  File "mrtarget/common/Redis.py", line 300, in _inner_run
    job_results = self.process(data)
  File "mrtarget/modules/Association.py", line 329, in process
    available_evidence = self.es_query.count_evidence_for_target(target)
  File "mrtarget/common/ElasticsearchQuery.py", line 448, in count_evidence_for_target
    'size': 0
  File "/usr/local/lib/python2.7/site-packages/elasticsearch/client/utils.py", line 73, in _wrapped
    return func(*args, params=params, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/elasticsearch/client/__init__.py", line 632, in search
    doc_type, '_search'), params=params, body=body)
  File "/usr/local/lib/python2.7/site-packages/elasticsearch/transport.py", line 312, in perform_request
    status, headers, data = connection.perform_request(method, url, params, body, ignore=ignore, timeout=timeout)
  File "/usr/local/lib/python2.7/site-packages/elasticsearch/connection/http_requests.py", line 90, in perform_request
    self._raise_error(response.status_code, raw_data)
  File "/usr/local/lib/python2.7/site-packages/elasticsearch/connection/base.py", line 125, in _raise_error
    raise HTTP_EXCEPTIONS.get(status_code, TransportError)(status_code, error_message, additional_info)
RequestError: TransportError(400, u'illegal_argument_exception', u'invalid version format: 
"CATEGORIES": {"CLINICAL_PRECEDENCE": 1.0, "DISCOVERY_PRECEDENCE": 0.0, "PREDICTED_TRACTABLE": 0.0}, "ENSEMBLE": -0.73153363, "HIGH_QUALITY_COMPOUNDS": 0, "SMALL_MOLECULE_GENOME_MEMBER": FALSE, "TOP_CATEGORY": "CLINICAL_PRECEDENCE"}}}}')
2019-03-12 17:29:54,841 mrtarget.common.ElasticsearchLoader WARNING - push to elasticsearch failed for chunk.  retrying in 5s...
```

**Comment by afaulconbridge** [comment]
No, I think it refering to the "version" of the document in Elasticseach - usually something used to avoid dropped updates. That's still a problem because I don't think the association step should be doing any updates to documents! Was this with public 19.02 data? Can we reproduce it?

**Comment by pieterlukasse** [comment]
Found this one as well:

```
2019-03-13 11:05:24,219 mrtarget.common.Redis DEBUG - ========> Processing evidence: target[ENSG00000185527], disease[EFO_0000311]
2019-03-13 11:05:24,279 mrtarget.common.ElasticsearchLoader WARNING - push to elasticsearch failed for chunk.  retrying in 5s...
2019-03-13 11:05:24,668 mrtarget.common.Redis ERROR - Error processing job 1870319318ba412aa8d5f94c44cf4d15: hits
Traceback (most recent call last):
  File "mrtarget/common/Redis.py", line 300, in _inner_run
    job_results = self.process(data)
  File "mrtarget/modules/Association.py", line 329, in process
    available_evidence = self.es_query.count_evidence_for_target(target)
  File "mrtarget/common/ElasticsearchQuery.py", line 451, in count_evidence_for_target
    return res['hits']['total']
KeyError: 'hits'
2019-03-13 11:05:24,696 mrtarget.common.Redis DEBUG - ############# found 64 evidences for target ENSG00000113721 
2019-03-13 11:05:24,821 mrtarget.common.Redis DEBUG - ############# found 1 evidences for target ENSG00000154678 
```

**Comment by pieterlukasse** [comment]
@afaulconbridge as agreed, moved second issue to a separate ticket: https://github.com/opentargets/platform/issues/482

**Comment by afaulconbridge** [comment]
I've reviewed our logfiles for and seen no evidence of this problem. Therefore I expect it to be resolved with 19.04. I'll leave this ticket open for now, and it should be re-examined with 19.04 to ensure the problem is solved.

I wonder if the underlying cause was the an issue similar to opentargets/platform#177 with fork() affecting file descriptors.

**Comment by pieterlukasse** [comment]
will assume to be solved in 19.04 as I have not seen this error while running 19.04.5 on my latest test set :+1:
